docs/basic_usage.rst

```diff
@@ -83,6 +83,12 @@ If the measured data that we are comparing with our model comes in at irregular
    9.5     112.541515
    …
 
+Retrieving totally flat dataframe
+---------------------------------
+The subscripted variables, in general, will be returned as *xarray.DataArray*s in the output *pandas.DataFrame*. To get a totally flat dataframe, like Vensim outuput the `flatten=True` when calling the run function::
+
+   >>> model.run(flatten=True)
```

Comment:
Is the default return behavior different from what it was before?

Reply:
No, the default is flatten=False and will return a DataFrame with xarrays unless the variable names with coordinates are passed. So the default behaviour is exactly as before.